Scripts/Source/minai_Sex.psc

```diff
@@ -1393,7 +808,7 @@ function onSexStart(int ThreadID, string framework)
     actors = slf.GetController(ThreadID).Positions
   endif
 
-  JMap.setForm(actorToSayOnEndMap, ThreadID, getWeightedRandomActorToSpeak(actors))
+  JMap.setForm(actorToSayOnEndMap, ThreadID, minai_SexUtil.GetWeightedRandomActorToSpeak(actors, PlayerRef, config, bHasOstim))
```

Comment:
Let's attach the utility scripts to a quest so that we can have persistent variables to avoid having to pass playerRef, config, etc in constantly.

Reply:
attached all scripts to main quest. Added one more temporary esp file to avoid conflicts(will merge it before PR is ready to merge)